app/ai/voice/agents/breeze_buddy/agent.py

```
@@ -562,12 +635,12 @@ async def run_pipeline():
 
-            with tracer.start_as_current_span(conversation_id) as root_span:
+            with tracer.start_as_current_span(self.conversation_id) as root_span:
                 # Store root span reference for updating with evaluation data later
                 self.root_span = root_span
 
                 logger.info(
-                    f"Starting Langfuse trace for Breeze Buddy conversation: {conversation_id}"
+                    f"Starting Langfuse trace for Breeze Buddy conversation: {self.conversation_id}"
                 )
 
                 root_span.set_attribute("conversation.id", conversation_id)
```

Ruff (0.14.10)

Undefined name `conversation_id`

(F821)